splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/BaseFormView.jsx

```diff
@@ -143,37 +180,39 @@ class BaseFormView extends Component {
         }
     }
 
+    // Set error message
     setErrorMsg = (msg) =>{
         const newFields = { ...this.state };
         newFields.ErrorMsg = msg;
         this.setState(newFields);
     }
 
-
-    // Not tested yet
-    clearAllErrorMsg = () =>{
-        const newFields = { ...this.state };
+    // Clear error message and errors from fields 
+    clearAllErrorMsg = (State) =>{        
+        const newFields = State ? { ...State } : {...this.state};
         newFields.ErrorMsg = "";
-        const newData = {...this.state.data}
-     
-        Object.keys(newData).map( (key) => {
+        const newData = State ? { ...State.data } : {...this.state.data};
+        const temData ={}
+        Object.keys(newData).forEach( (key) => {
             if(newData[key].error){
                 const tem = {...newData[key]}
                 tem.error = false;
-                return tem;
+                temData[key] = tem;
```

Comment:
please make the variable names readable. if its temporary then name it `temp`. It can be avoided as well in this case.